articles/forward-and-backward-chaining-in-ai/index.md

```diff
@@ -0,0 +1,119 @@
+## Forward and Backward Chaining in Artificial Intelligence
+
+Backward and forward chaining are methods of reasoning that exist in the Expert System Domain of artificial intelligence. 

```

Comment:
The introduction is needed. Why should one read this article? Give a solid intro to what we are trying to achieve and what the reader will know at the end of the article. It is important to grad the attention of the reader.

Reply:
Done.